src/check/parse/Parser.zig

```diff
@@ -161,25 +161,11 @@ pub fn parseFile(self: *Parser) void {
     });
 }
 
-fn parseCollection(self: *Parser, comptime T: type, end_token: Token.Tag, scratch: *std.ArrayListUnmanaged(T), parser: fn (*Parser) T) ExpectError!usize {
-    const scratch_top = scratch.items.len;
-    while (self.peek() != end_token and self.peek() != .EndOfFile) {
-        scratch.append(self.gpa, parser(self)) catch |err| exitOnOom(err);
-        self.expect(.Comma) catch {
-            break;
-        };
-    }
-    self.expect(end_token) catch {
-        return ExpectError.expected_not_found;
-    };
-    return scratch_top;
-}
-
```

Comment:
This wasn't being used anywhere. I think it just didn't get removed when we switched to Spans everywhere

Reply:
Yeah, I think it got replaced by `parseCollectionSpan`